### PR TITLE
Added the rating field to the BUO.

### PR DESCRIPTION
**Pull request by E-B-Smith**

### CODE REVIEW

**Comment by BranchMacMini** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/qabot.stage.branch.io/d318cfeecc98d7b4fdff9a6e0f431959/d318cfeecc98d7b4fdff9a6e0f431a5b)
---
*Duration*: 14 minutes and 42 seconds
*Result*: All 224 tests passed, but please **fix 3 warnings**.
*Test Coverage*: 62%

File: Branch-TestBed/Branch-SDK-Tests/BranchEvent.Test.swift [hunk 1]
```diff
@@ -44,14 +44,18 @@ class BranchEventTestSwift : BNCTestCase {
...
+        branchUniversalObject.contentMetadata.rating            = 6;
         branchUniversalObject.contentMetadata.addressStreet     = "Street_name1"
         branchUniversalObject.contentMetadata.addressCity       = "city1"
         branchUniversalObject.contentMetadata.addressRegion     = "Region1"
         branchUniversalObject.contentMetadata.addressCountry    = "Country1"
         branchUniversalObject.contentMetadata.addressPostalCode = "postal_code"
         branchUniversalObject.contentMetadata.latitude          = 12.07
         branchUniversalObject.contentMetadata.longitude         = -97.5
-        branchUniversalObject.contentMetadata.imageCaptions     = [ "my_img_caption1",  "my_img_caption_2"]
+        branchUniversalObject.contentMetadata.imageCaptions     = [
```
Comment (sojanpr) [inline Branch-TestBed/Branch-SDK-Tests/BranchEvent.Test.swift]: Nitpick: are  these format changes  intentional

**Comment by sojanpr** [comment]
LGTM 👍

File: Branch-TestBed/Branch-SDK-Tests/BranchEvent.Test.swift (hunk 1 above)
Comment (E-B-Smith) [reply Branch-TestBed/Branch-SDK-Tests/BranchEvent.Test.swift]: Yes.

**Pull request merged**